.github/workflows/stale.yml

```diff
@@ -27,8 +27,7 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         days-before-stale: 180
         days-before-close: 14
-        stale-pr-label: 'stale'
```

Comment:
Why is this removed as well?  Feels unrelated to the addition of `exempt-issue-labels`?

Reply:
We have two different configurations here: one for issues the other for PRs.

This section here is for the issue, and has the appropriate `stale-issue-label`. The `stale-pr-label` is a useless setting in this context.